contracts/cw721-non-transferable/src/msg.rs

```diff
@@ -0,0 +1,10 @@
+use cosmwasm_schema::cw_serde;
+
+use cosmwasm_std::Addr;
+use cw721_base::InstantiateMsg as Cw721InstantiateMsg;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub admin: Option<Addr>,
+    pub cw721_instantiate_msg: Cw721InstantiateMsg,
```

Comment:
May be better UX for frontend devs if you just inline the `Cw721InstantiateMsg` here? That way, could for existing NFT minting would _just work_ as the `admin` field is optional.

Reply:
makes sense! - comment addressed